### DISCUSSION TITLE
Prevent boat from sinking to the ground

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1775] [Help]
Hey, when I summon my boat on water, it will drop to the ground. How can I prevent that my boat will "sink"?

**Comment by zax71** [comment]
Iirc Minestom does not implement water physics. You will have to look at Vanilla Reimplementation for how water physics work.

**Comment by FluxCapacitor2** [comment]
The easiest way to fix this would be to [disable gravity](https://javadoc.minestom.net/net/minestom/server/entity/Entity.html#setNoGravity(boolean)) on the boat:
```java
boatEntity.setNoGravity(true);
```
